### ISSUE TITLE
May I ask how to use black, white and red e-paper to display only black and white？

### ISSUE THREAD

**Issue by bsqzfh** [robweber/vsmp-plus#11]
I am using the Weixue 7.5inch e-Paper HAT B V2 screen, because it is currently very cheap in my place, but I only want it to display black and white, the red color makes the picture very dark, the contrast is also high, and it looks ugly. Is there any way to do it?

**Comment by robweber** [comment]
Yes, there should be. You're using this using the `waveshare_epd.epd7in5b` device name right? The omni-epd package allows for some options that can be passed to the display. In this case you'll want to set the color mode to __bw__. You'll need to create a file called `omni-epd.ini`. Instructions can be found on the [omni-epd page](https://github.com/robweber/omni-epd#advanced-epd-control). You can also adjust contrast and brightness, which may help the viewing as well.
